botCommands/points.js

```diff
@@ -91,6 +91,12 @@ function exclamation(points, isGoodQuestion) {
   if (points === 4000) {
     return '`//TODO: Implement Club 4000`';
   }
+  if (points === 5000) {
+    return 'FIVE THOUSAND POINTS. Now go touch grass.';
```

Comment:
```suggestion
   return 'FIVE THOUSAND POINTS, [HALF WAY THERE](https://youtu.be/lDK9QqIzhwk?t=93)'
```

Reply:
Nit and also prevent the youtube embed from popping up? Or intentional?
```suggestion
    return 'FIVE THOUSAND POINTS, [WE'RE HALF WAY THERE!](<https://youtu.be/lDK9QqIzhwk?t=93>)'
```

Reply:
That being said, why halfway there for 5k? Considering you may change 10k to quarter of the way there (makes sense for 40k)

If Bon Jovi should be anywhere, I'd say it needs to be 20 (and only 20)

Reply:
> That being said, why halfway there for 5k? Considering you may change 10k to quarter of the way there (makes sense for 40k)

Because 10K is also a nice milestone, and tbh I'm counting on people forgetting what the 5K text was by the time they hit 10K

> If Bon Jovi should be anywhere, I'd say it needs to be 20 (and only 20)

I thought about it, but I feel like that might enforce a feeling of 40 points as like a goal to reach (with the associated club-40), rather than something that should just happen as a result of making good contributions to the Discord.

Reply:
Fair on the point about 20. 5k being "halfway" (given the 10k message) still feels a little odd to me, though that's only my opinion and not blocking of course.

Reply:
> Fair on the point about 20. 5k being "halfway" (given the 10k message) still feels a little odd to me, though that's only my opinion and not blocking of course.

I get that, and I partially agree, but what I'm basically hoping for is that at some point A someone will get to 5K, go huh that's true, another 5K and I'm up to the next order of the magnitude, then at some point B will get to 10K, go huh that's a fun club-40 joke, and that point A and B will be far enough apart nobody will notice the discrepancy

Reply:
@Asartea, I think you are getting warmer. These messages are definitely more celebratory in tone.

But I think for anyone earning 5k+ points, represents a sincere and long-term commitment on the part of the receiver to helping innumerable learners on their journey, and I'd think that the messages beyond even 1k (I know this PR is just for 5k points and 10k points) could/should be a message that would communicate thanks in a celebratory/congratulatory tone. Each is a milestone more than worthy for resting at, rather than a race to forty thousand points. 

In a way, I like your initial suggestions more than the rewrites. They just could be more celebratory/congratulatory in nature. 

Do you want to take another crack at the 5k and 10k messages?